smoke-tests/spec/kubernetes_helper.rb

```diff
@@ -69,6 +69,24 @@ def object_exists?(namespace, type, name)
   status.success?
 end
 
+# Creates a deplyoment using the postgres image with a privileged container.
+def create_privileged_deploy(namespace)
+  apply_template_file(
+    namespace: namespace,
+    file: "spec/fixtures/privileged-deployment.yaml.erb",
+    binding: binding
+  )
+end
+
+# Creates a deployment using the bitnami/nginx image with a unprivileged container.
+def create_unprivileged_deploy(namespace)
+  apply_template_file(
+    namespace: namespace,
+    file: "spec/fixtures/unprivileged-deployment.yaml.erb",
+    binding: binding
+  )
+end
```

Comment:
These seem very specific to the PSP tests, so I'd probably put them inline in the psp_spec.rb file. But, since the only difference is the name of the ERB file, and the name includes "privileged" and "unprivileged", I'm not sure it's worth defining methods for these.

Reply:
I agree with this, I've now moved it to the psp spec and made the function generic.
```
# Creates a deplyoment using the arguments defined
def create_psp_deployment(namespace, deployment_file)
  apply_template_file(
    namespace: namespace,
    file: deployment_file,
    binding: binding
  )
end

```